data/schema.rdfa

```diff
@@ -11039,6 +11032,171 @@ Typical unit code(s): C62 for person</span>
+    <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/HowToTool">HowToTool</a></span>
+</div>
+
+<div typeof="rdf:Property" resource="http://schema.org/totalTime">
+    <span class="h" property="rdfs:label">totalTime</span>
+    <span property="rdfs:comment">The total time required to perform instructions or a direction (including time to prepare the supplies), in [ISO 8601 duration format](http://en.wikipedia.org/wiki/ISO_8601).</span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/HowTo">HowTo</a></span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/HowToDirection">HowToDirection</a></span>
+    <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Duration">Duration</a></span>
+</div>
+
+<div typeof="rdf:Property" resource="http://schema.org/yield">
+    <span class="h" property="rdfs:label">yield</span>
+    <span property="rdfs:comment">The quantity that results by performing instructions. For example, a paper airplane, 10 personalized candles.</span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/HowTo">HowTo</a></span>
+    <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/QuantitativeValue">QuantitativeValue</a></span>
+</div>
+
+<h1>HowToDirection</h1>
+
+<div typeof="rdfs:Class" resource="http://schema.org/HowToDirection">
+    <span class="h" property="rdfs:label">HowToDirection</span>
+    <span property="rdfs:comment">A direction indicating a single action to do in the instructions for how to achieve a result.</span>
+    <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/ListItem">ListItem</a></span>
+</div>
+
+<div typeof="rdf:Property" resource="http://schema.org/afterMedia">
+    <span class="h" property="rdfs:label">afterMedia</span>
+    <span property="rdfs:comment">A media object representing the circumstances after performing this direction.</span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/HowToDirection">HowToDirection</a></span>
+    <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/MediaObject">MediaObject</a></span>
+</div>
+
+<div typeof="rdf:Property" resource="http://schema.org/beforeMedia">
+    <span class="h" property="rdfs:label">beforeMedia</span>
+    <span property="rdfs:comment">A media object representing the circumstances before performing this direction.</span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/HowToDirection">HowToDirection</a></span>
+    <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/MediaObject">MediaObject</a></span>
+</div>
```

Comment:
Just to clarify, they don't apply to the steps as a whole, they apply to a single direction in a single step.

Many instructions have multiple pictures for some of the steps (e.g. http://www.instructables.com/id/How-to-Make-a-Modern-Wooden-Clock/). However, the temporal connections between the pictures and the action are not specified (or specifiable). beforeMedia, duringMedia, and afterMedia solve this by enabling the author to annotate how things should look "before", "during" and "after" an action, respectively.